jdaviz/configs/imviz/plugins/parsers.py

```diff
@@ -124,6 +125,15 @@ def _parse_image(app, file_obj, data_label, ext=None):
     data_iter = get_image_data_iterator(app, file_obj, data_label, ext=ext)
 
     for data, data_label in data_iter:
+        if data.coords is not None:
+            if isinstance(data.coords, GWCS) and (data.coords.bounding_box is not None):
```

Comment:
Now that I look at this again, I don't think we need the first `if`.

Reply:
that's probably true, is it worth changing here or maybe just in any PR to main?

Reply:
I can PR to main. This here is consistent with the thing being backported.